### PR TITLE
Landing page responsiveness

### PR DESCRIPTION
**Pull request by w41eed**
Landing Page should now be responsive without any overlapping or overflowing.
1) Fixed the overlapping of feature cards when resizing
2) Fixed disappearing senior executive section and overlapping of images when resizing
3) Fixed black tint in the newsletter section from overflowing into the about section
4) Fixed Main Image buttons from overflowing into features section

### CODE REVIEW

File: src/Components/LandingPage/SeniorExecGrid.js [hunk 1]
```diff
@@ -3,12 +3,12 @@ import Grid from "@material-ui/core/Grid";
...
 import { makeStyles } from "@material-ui/core/styles";
 import SeniorExecText from "./SeniorExecText";
-import SmallSeniorExec from "./SmallSeniorExec";
+// import SmallSeniorExec from "./SmallSeniorExec";
 
 // import SmallSeniorExecImgHover from "../Images/senior/Latafat_Faran2.jpg";
 import SmallSeniorExecImg from "../Images/senior/Jalaluddin_Uzma.jpg";
 import SmallSeniorExecImg2 from "../Images/senior/Diop_Cheikh.jpeg";
-import SmallSeniorExecBottom from "./SeniorExecSmallBottom";
+// import SmallSeniorExecBottom from "./SeniorExecSmallBottom";
```
Comment (ahmedhamodi) [inline src/Components/LandingPage/SeniorExecGrid.js:11]: Are these supposed to be commented out?

File: src/Components/LandingPage/SeniorExecGrid.js [hunk 2]
```diff
@@ -125,14 +126,14 @@ class SeniorExecGrid extends Component {
...
-            <Grid
+            {/* <Grid
               container
               item
               xs={8}
               spacing={0}
               alignItems="center"
               justify="center"
-            >
+            > */}
```
Comment (ahmedhamodi) [inline src/Components/LandingPage/SeniorExecGrid.js:136]: Is this supposed to be commented out? If so, you can probably just remove it.

File: src/Components/LandingPage/SeniorExecGrid.js [hunk 3]
```diff
@@ -194,11 +215,11 @@ class SeniorExecGrid extends Component {
                 image={this.state.displayImages[3].photo}
                 hover_image={this.state.displayImages[3].photo}
               />
-            </Grid>
+            </Grid> */}
```
Comment (ahmedhamodi) [inline src/Components/LandingPage/SeniorExecGrid.js:218]: Uncomment or remove?

File: src/Components/LandingPage/SeniorExecGrid.js [hunk 4]
```diff
@@ -264,7 +285,7 @@ class SeniorExecGrid extends Component {
             </Grid>
             <SeniorExecText appContext={this.props.appContext} />
           </Grid>
-        </div>
+        </div> */}
```
Comment (ahmedhamodi) [inline src/Components/LandingPage/SeniorExecGrid.js:288]: Uncomment or remove?

**Review by ahmedhamodi — COMMENTED** [review]
Quite a bit of commented out code, wanted to make sure it was intentional or not. I will run and test the code changes visually later tonight.

File: src/Components/LandingPage/SeniorExecGrid.js (hunk 1 above)
Comment (w41eed) [reply src/Components/LandingPage/SeniorExecGrid.js]: Oh sorry, i forgot to remove all the commented out code, my bad!

**Review by ahmedhamodi — APPROVED** [review]
Overall this is a major improvement from what we had previously. So I am okay if you merge this and address the following comments with a different PR. But there are 2 issues I am noticing.

On a screen size of around 1060px width, the buttons get cut off as follows:
image

On a iPhone 5 (which idk if we should even build for anymore, since it's old and way too small), but I get this issue with the buttons (button overflow):
image

Overall, great work. Thanks for doing this.

**Review by arsidada — APPROVED** [review]

**Comment by arsidada** [comment]
I'm also ok with merging this and carrying over the remaining 2 issues in another issue. At this point they are too minor. I've created an issue for the 2 issues @ahmedhamodi pointed out here: https://github.com/maxgala/aspire-frontend/issues/203.

Going to merge this.

**Pull request merged**